eZ/Publish/Core/MVC/Symfony/Controller/Content/QueryController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
```

Comment:
I guess we haven't fully (re)defined this, but I would have thought heading should be:

```php
<?php

 * File containing the LanguageHandler class.
 *
 * @copyright Copyright (C) eZ Systems AS. All rights reserved.
 * @license For full copyright and license information view LICENSE file distributed with this source code.
 */
```

Ideally skipping the file description in favor of rather spending the effort by having more useful class description, including usage examples.